components/Alert/index.jsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+
+export const alertTestIds = {
+  icon: 'alert-icon',
+  title: 'alert-title',
+  description: 'alert-description',
+};
+
+const Alert = ({ icon, title, description }) => {
+  return (
+    <div className="w-full bg-notification backdrop-blur-2.5xl flex p-4 rounded-xl">
+      <span data-test-id={alertTestIds.icon}>
+        <Image src={icon} alt="alert" />
+      </span>
+      <div className="text-white ml-5">
```

Comment:
Should this be `ml-4` ?

```suggestion
      <div className="text-white ml-4">
```

Current style | The design
--- | ---
![Screen Shot 2565-03-31 at 11 22 24](https://user-images.githubusercontent.com/1772999/160976007-5ddb2689-c969-4335-985b-1f8966cc97b0.png) | ![Screen Shot 2565-03-31 at 11 18 51](https://user-images.githubusercontent.com/1772999/160976023-4e2c7ffd-a24e-41b6-a858-c6458c451963.png)

Reply:
done in 98bcf8c4034b7b2509d56f4680084e3416499844